src/PasswordStorage/ManagePassword.java

```diff
@@ -0,0 +1,118 @@
+        //Scanner scanner = new Scanner(System.in);
+
+        // get credentials
+        //System.out.println("Enter a Title for the credentials:");
+        //System.out.println("Enter a Username");
+        //String username = scanner.nextLine();
+        //System.out.println("Enter a Password:");
+        //String password = scanner.nextLine();
+
+        //return scanner.nextLine();
+    
+    //}
+
+    // Used to write a new Password File at the moment - Append to come soon
+    // Add "ManagePassword.WritePassword();" to main to execute.
+    public static void WritePassword() {
+
+        try {
+            Scanner scanner = new Scanner(System.in);
+
+            // horrible way of getting data
+            System.out.println("Enter a Title for the credentials:");
+            String title = scanner.nextLine();
+            System.out.println("Enter a Username");
+            String username = scanner.nextLine();
+            System.out.println("Enter a Password:");
+            String password = scanner.nextLine();
+
+            // Changed the directory to desktop for easy testing, if not specified it
+            // will print in project folder
+            // Change the directory to make sense for you
+            BufferedWriter credWriter = new BufferedWriter(new FileWriter(
+                    //"password,txt"));
+                    "/home/dannymac/Desktop/test.txt"));
+
+
+            // NOTE!!! Password will be plaintext for now, will work on encryption soon.
+            // DO NOT STORE ANY MEANINGFUL PASSWORD AT THE MOMENT
+            String credData = ("Title: " + title + "\nUsername: " + username + "\nPassword: " + password);
+            credWriter.write(credData);
+            // clear memory and close writer
+            credWriter.flush();
+            credWriter.close();
+        }
+        // Catch the exception
+        catch (IOException e) {
+
+            System.out.println(e.getMessage());
+        }
+    }
+
+    public static void ReadPassword() {
+
+        try {
+            // Changed the directory to desktop for easy testing, if not specified it
+            // will print in project folder
+
+            // Change the directory to make sense for you
+            BufferedReader credReader = new BufferedReader(new FileReader(
+                    //"password,txt"));
+                    "/home/dannymac/Desktop/test.txt"));
+
+
+            // NOTE!!! Password will be plaintext for now, will work on encryption soon.
+            // DO NOT STORE ANY MEANINGFUL PASSWORD AT THE MOMENT
+            String data;
+            while ((data = credReader.readLine()) !=null) {
+                System.out.println(data);
+            }
+        }
+        // Catch the exception
+        catch (IOException e) {
+
+            System.out.println(e.getMessage());
+        }
+
+    }
+
+    public static void AppendPassword() {
+
+        String credFile = "/home/dannymac/Desktop/test.txt";
```

Comment:
Need to add a way to either set a default path or a user defined path, likely to be implemented in the user interface classes.

Reply:
Made credFile a variable that can be changed. We will need to make another class that takes input from the user and sets the value to allow users to change between different credential files.